lib/widgets/message_list.dart

```diff
@@ -1015,10 +1018,12 @@ class StreamMessageRecipientHeader extends StatelessWidget {
     super.key,
     required this.message,
     required this.showStream,
+    required this.isTopicNarrow,
   });
 
   final StreamMessage message;
   final bool showStream;
+  final bool isTopicNarrow;
```

Comment:
nit: Let's call this `inTopicNarrow`; it's not meant to be about _what_ the recipient header is, but _where_ it is.